gapless5.js

```diff
@@ -186,7 +186,7 @@ function Gapless5Source(parentPlayer, parentLog, inAudioPath) {
     } else if ((audio !== null) && (queuedState === Gapless5State.None) && this.inPlayState(true)) {
       log.debug(`Switching from HTML5 to WebAudio: ${this.audioPath}`);
       setState(Gapless5State.Stop);
-      this.play(true, true);
+      this.play(true, false, true);
```

Comment:
How come `skipCallback` changed from true to false here?  What are the repercussions of this?